apps/remix-ide/src/migrateFileSystem.js

```diff
@@ -53,7 +53,11 @@ const populateWorkspace = async (workspace, json, browserProvider) => {
       browserProvider.createDir(joinPath(workspace, item))
       await populateWorkspace(workspace, json[item].children, browserProvider)
     } else {
-      await browserProvider.set(joinPath(workspace, item), json[item].content)
+      await browserProvider.set(joinPath(workspace, item), json[item].content, (err) => {
```

Comment:
what is the typical message that is returned when the local storage is full?

Reply:
![Screenshot 2021-04-06 at 12 13 48](https://user-images.githubusercontent.com/2787300/113695871-a5449a80-96d1-11eb-93e1-048605bd608c.png)

Reply:
is ti possible to stop the migration if that happens?

Reply:
it's a recursive script... the old files just stay there right? It can capture any error, it could also be the storage is not full when migration starts. If storage is full files get created but they will be empty if the result would surpass the storage limit.

Reply:
So? Is it good enough or not? @yann300